serverless.js

```diff
@@ -1,23 +1,13 @@
-const { mkdirSync, existsSync, unlinkSync, symlinkSync } = require('fs');
-const { join, dirname } = require('path');
 const { paramCase } = require('param-case');
 const pkg = require('./package.json');
 
-const service = paramCase(pkg.name);
-const plugins = ['serverless-s3-sync'];
+const { NODE_ENV = 'production' } = process.env;
```

Comment:
I would argue it's safer to have nothing default to production and only CI master builds set things to prod